engine-rest/engine-rest-openapi/src/main/templates/paths/history/process-instance/delete/post.ftl

```diff
@@ -0,0 +1,57 @@
+{
+  <@lib.endpointInfo
+      id = "deleteHistoricProcessInstancesAsync"
+      tag = "Historic Process Instance"
+      summary = "Delete Async (POST)"
+      desc = "Delete multiple historic process instances asynchronously (batch).
+              At least `historicProcessInstanceIds` or `historicProcessInstanceQuery` has to be provided.
+              If both are provided then all instances matching query criterion and instances from the list will be deleted." />
+
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "DeleteHistoricProcessInstancesDto"
+      examples = [
+                  '"example-1": {
+                     "summary": "POST `/history/process-instance/delete`",
+                     "value": {
+                                "deleteReason" : "aReason",
+                                "historicProcessInstanceIds": ["aProcess","secondProcess"],
+                                "historicProcessInstanceQuery": {
+                                  "startedAfter": "2016-10-11T11:44:13.000+0200",
+                                  "finishedBefore": "2016-10-13T11:44:17.000+0200"
+                                }
+                              }
+                   }'
+                ] />
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        dto = "BatchDto"
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "Status 200 response",
+                       "description": "Response for POST `/history/process-instance/delete`",
+                       "value": {
+                                  "id": "120b568d-724a-11e9-98b7-be5e0f7575b7",
+                                  "type": "process-set-removal-time",
+                                  "totalJobs": 12,
+                                  "batchJobsPerSeed": 100,
+                                  "invocationsPerBatchJob": 1,
+                                  "seedJobDefinitionId": "120b5690-724a-11e9-98b7-be5e0f7575b7",
+                                  "monitorJobDefinitionId": "120b568f-724a-11e9-98b7-be5e0f7575b7",
+                                  "batchJobDefinitionId": "120b568e-724a-11e9-98b7-be5e0f7575b7",
+                                  "tenantId": "accounting"
```

Comment:
suspended and createUserId are missing in batch example:
```suggestion
                                  "tenantId": "accounting", 
                                  "suspended": false,
                                  "createUserId": null
```

Reply:
👍